### PR TITLE
Update README and fix a bug in off-device keystore usecase

### PR DESCRIPTION
**Pull request by desokroshan**

### CODE REVIEW

File: AndroidPubSub/src/com/amazonaws/demo/androidpubsub/PubSubActivity.java [hunk 1]
```diff
@@ -235,7 +235,12 @@ void initIoTClient() {
                     // load keystore from file into memory to pass on connection
                     clientKeyStore = AWSIotKeystoreHelper.getIotKeystore(certificateId,
                             keystorePath, keystoreName, keystorePassword);
-                    btnConnect.setEnabled(true);
+                    runOnUiThread(new Runnable() {
```
Comment (kvasukib) [inline AndroidPubSub/src/com/amazonaws/demo/androidpubsub/PubSubActivity.java]: WHy is this required?

Comment (desokroshan) [reply AndroidPubSub/src/com/amazonaws/demo/androidpubsub/PubSubActivity.java]: The button update code is on a background thread since `initIoTClient ` is called from the AWSMobileClient initialization success callback.

Comment (kvasukib) [reply AndroidPubSub/src/com/amazonaws/demo/androidpubsub/PubSubActivity.java]: Just add a comment on why we are doing this.
